### DISCUSSION TITLE
Renovate Repository Terraform Lock File Maintenance?

### DISCUSSION THREAD

**Discussion by sixdaysandy** [renovatebot/renovate#18539] [Request Help]
Hi,

Is there a way to enable the terraform lockfile maintenance for the github app version of Renovate?
I can see that it should be built into the self-hosted version, but I can't find any examples of it working for the repository version?

In our repo it's just updating the versions block but not performing the lockfile maintenance.

Thanks,
Andy

**Comment by viceice** [comment]
Terraform lockfile maintenance is fully supported through hosted app.
Please check the debug logs when renovate has pushed the commit.
